lib/reporter.js

```diff
@@ -35,89 +28,74 @@ var createErrorFormatter = function (config, emitter, SourceMapConsumer) {
     '(\\:(\\d+))?' + // column
     '', 'g')
 
-  var getSourceMapConsumer = (function () {
-    var cache = new WeakMap()
-    return function (sourceMap) {
-      if (!cache.has(sourceMap)) {
-        cache.set(sourceMap, new SourceMapConsumer(sourceMap))
-      }
-      return cache.get(sourceMap)
+  const cache = new WeakMap()
+
+  function getSourceMapConsumer (sourceMap) {
+    if (!cache.has(sourceMap)) {
+      cache.set(sourceMap, new SourceMapConsumer(sourceMap))
     }
-  }())
+    return cache.get(sourceMap)
+  }
+
+  function formatFullPath (path, line, column) {
+    return path + (line ? `:${line}` : '') + (column ? `:${column}` : '')
+  }
 
   return function (input, indentation) {
-    indentation = _.isString(indentation) ? indentation : ''
+    indentation = helper.isString(indentation) ? indentation : ''
```

Comment:
Rather than introduce `helper` just for this one case while keeping `lodash` for the  other cases, I'd rather consistently use one or the other.

Reply:
Fixed